app/components/navbar/navbar_component.html.erb

```diff
@@ -12,14 +8,19 @@
             <% end %>
             <%= nav_link("Find an event near you", events_path) %>
         </ul>
+
+        <div class="navbar__desktop__search">
+          <%= link_to image_pack_tag("icon-site-search.svg"), search_path unless Rails.env.production? %>
+        </div>
```

Comment:
Should the `unless` statement wrap the whole `div` here? Not sure it will matter much.

Reply:
Yeh - you're right, that was a last minute change - I'll update it